src/game.cpp

```diff
@@ -9652,6 +9652,12 @@ bool game::phasing_move( const tripoint &dest_loc )
            ( critter_at( dest ) != nullptr && tunneldist > 0 ) ) {
         //add 1 to tunnel distance for each impassable tile in the line
         tunneldist += 1;
+        //Being dimensionally anchored prevents quantum shenanigans.
+        if( u.worn_with_flag( "DIMENSIONAL_ANCHOR" ) || u.has_effect_with_flag( "DIMENSIONAL_ANCHOR" ) ) {
+            u.add_msg_if_player( m_info, _( "You are repeled by the barrier!" ) );
+            u.charge_power( -250_kJ ); //cost of tunneling one tile.
```

Comment:
this function was migrated recently. 
```suggestion
            u.mod_power_level( -250_kJ ); //cost of tunneling one tile.
```
you may need to rebase on current master.